tests/test_internals/test_si.py

```diff
@@ -585,66 +585,56 @@ def test_group_set_custom_description(self):
                     assert nwb.electrodes["group"][i].description == "M1 description"
 
 
-class TestSpikeInterfaceRecorders(unittest.TestCase):
+class TestAddElectrodes(unittest.TestCase):
```

Comment:
I think this is way more descriptive of what the test is doing right now. I was thinking that we would like to add more `spikeinterface` specific functionality that we would like to test in the future and decided to leave it more generally. Maybe this is an instance of early optimization (early generalization?). Note also that there is a test called `TestWriteElectrodes` above leading to some semantic overlap.